cfgov/v1/jinja2/v1/_includes/organisms/well.html

```diff
@@ -0,0 +1,22 @@
+
+{# ==========================================================================
+
+   o_well.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create a grey well organism.
+   See [GHE]/flapjack/Modules-V1/wiki/Well
+
+   template: Template URL for content to include.
+
+   ========================================================================== #}
+
+{% macro render(template) %}
+{% import template as content %}
```

Comment:
Are you meaning the value of `template`? Currently it is a URL. Can we pass in an arbitrary string of HTML and do anything with it? I didn't try.